extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -195,9 +195,11 @@ const OHIFCornerstoneViewport = React.memo(props => {
             DisplaySetService
           );
 
+          const keepCamera = true;
```

Comment:
This should be done/handled by one of the temporal synchronizations - ideally making the regular synchronizer also be able to apply the temporal version of the same synchronization - otherwise the keepCamera setting is wrong for many viewports and will distort the view.